code/formatters.py

```diff
@@ -151,6 +152,7 @@ def every_word(word_func):
     "snake": formatters_dict["SNAKE_CASE"],
     "string": formatters_dict["SINGLE_QUOTED_STRING"],
     "title": formatters_dict["CAPITALIZE_ALL_WORDS"],
+    "abbreviate": formatters_dict["ABBREVIATION"],
```

Comment:
Either way, I wouldn't make it exposed as a formatter people can use, as they might as well just spell using the Talon alphabet